js/contextMenus.js

```diff
@@ -53,7 +53,7 @@ const addBookmarkMenuItem = (label, siteDetail, closestDestinationDetail, isPare
       if (isParent) {
         siteDetail = siteDetail.set('parentFolderId', closestDestinationDetail && (closestDestinationDetail.get('folderId') || closestDestinationDetail.get('parentFolderId')))
       }
-      windowActions.setBookmarkDetail(siteDetail, siteDetail, closestDestinationDetail, true)
```

Comment:
Should it only show the location if you already have the site bookmarked (similar to the star behavior)? Or always show the location when you cmd+d or select it from the menu?